chia/rpc/wallet_request_types.py

```diff
@@ -42,6 +42,109 @@ def default_raise() -> Any:  # pragma: no cover
     raise RuntimeError("This should be impossible to hit and is just for < 3.10 compatibility")
 
 
+@streamable
+@dataclass(frozen=True)
+class Empty(Streamable):
+    pass
+
+
+@streamable
+@dataclass(frozen=True)
+class LogIn(Streamable):
```

Comment:
Yes, but in a previous PR.  The lack of `Request` is an artifact of a precious iteration which used `Request` subclass so stuff looked like:
```
class Login(Request):
  ...
```
making the second hypothetical `Request` feel redundant.  Obviously, that iteration is no longer a thing so we might want to change it but looking forward to ergonomics something like:
```
client.do(Login(...))
```
I kind of like the simplicity of the just the endpoint name rather than the `Request` suffix.  In addition, I think we'll probably start just importing the package at some point and doing like `wallet_request_types.LogIn` which gives the information that way as well.

As verbose as that is, I don't care that much so if you feel strongly, I can make a backlog ticket to update the whole file in that way.

Reply:
Nope, not that important.  Though I'll reserve the right to care more at a later time.  Maybe.  Thanks.